.github/workflows/ci.yml

```diff
@@ -323,14 +326,24 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: '14.x'
+      - run: |
+          echo SERVERLESS_SERVICE=edu >> $GITHUB_ENV; 
+          if [[ $GITHUB_REF == $STAGING_BRANCH ]]; then
+            echo SERVERLESS_STAGE=staging >> $GITHUB_ENV;
+          elif [[ $GITHUB_REF == $PRODUCTION_BRANCH ]]; then
+            echo SERVERLESS_STAGE=production >> $GITHUB_ENV;
+          fi
       - name: Package Dockerrun.aws.json
         run: |
           sed -i -e "s|@REPO|$REPO|g" Dockerrun.aws.json
           sed -i -e "s|@TAG|$TAG|g" Dockerrun.aws.json
-          zip -r "deploy.zip" Dockerrun.aws.json
+          sed -i -e "s|@DD_API_KEY|$DD_API_KEY|g" -e "s|@DD_ENV|$DD_ENV|g" .ebextensions/99datadog.config
+          zip -r "deploy.zip" .ebextensions Dockerrun.aws.json
         env:
           REPO: ${{env.ECR_URL}}/${{env.ECR_REPO}}
           TAG: ${{ needs.build-edu.outputs.tag }}
+          DD_API_KEY: ${{ secrets.DD_API_KEY }}
+          DD_ENV: ${{ env.NODE_ENV }}
```

Comment:
<img width="610" alt="Screenshot 2022-07-25 at 12 04 16" src="https://user-images.githubusercontent.com/39231249/180752023-a48acda6-ce46-4bbf-80ed-47680f7bc3b3.png">

Belated review but there's a small error here, this resolves to null as there's no `NODE_ENV` variable defined in this VM.

Reply:
Should be `SERVERLESS_STAGE`

Reply:
Sure, I will fix in another MR 🌮